snuba/utils/schemas.py

```diff
@@ -702,3 +706,54 @@ def set_modifiers(self, modifiers: Optional[TModifiers]) -> Enum[TModifiers]:
 
     def get_raw(self) -> Enum[TModifiers]:
         return Enum(self.values)
+
+
+class InvalidColumnType(SerializableException):
```

Comment:
Very minor nit: maybe `InvalidColumnTypeException` is clearer, but not important